app/models.py

```diff
@@ -471,3 +486,8 @@ def filter_null_value_fields(obj):
     return dict(
         filter(lambda x: x[1] is not None, obj.items())
     )
+
+
+def generate_new_service_id():
+    # TODO: Decide what we want G7 service IDs to look like and implement
```

Comment:
The comment is still valid - the uuid4 is here as a placeholder until we resolve the issue.  @minglis was going to add a story to Pivotal about this.